class.photon.php

```diff
@@ -358,7 +358,7 @@ public static function filter_the_content( $content ) {
 					// WP Attachment ID, if uploaded to this site
 					if (
 						preg_match( '#class=["|\']?[^"\']*wp-image-([\d]+)[^"\']*["|\']?#i', $images['img_tag'][ $index ], $attachment_id ) &&
-						0 === strpos( $src, $upload_dir['baseurl'] ) &&
```

Comment:
Since we don't use it anymore in that function, maybe we could remove `$upload_dir = wp_get_upload_dir();` from `filter_the_content`?

Reply:
Nice catch! I've removed the call to `wp_get_upload_dir()`